quixstreams/models/topics/topic.py

```diff
@@ -160,6 +163,8 @@ def name(self) -> str:
 
     @property
     def config(self) -> TopicConfig:
+        if self._config is None:
+            raise TopicConfigUndefined("No TopicConfig was provided.")
```

Comment:
I think raising an exception here is not the right strategy.

We should do one of the two:
1. Either make the `config` param required in `__init__`. It's easier and makes more sense than raising an error whenever the `.config` property is accessed.
2. Or provide some default value for `config`.

The plain `Topic` object is used only in tests, and it's not supposed to be created directly outside of the lib.
So I think we need to do №1, i.e. make it a required non-optional parameter and update the tests.

Reply:
Yeah, did option 1 :+1: